### PR TITLE
HomeTask1

### PR DESCRIPTION
**Pull request by egar347**

### CODE REVIEW

**Comment by lizarusi** [comment]
Perfect!

**Review by lizarusi — APPROVED** [review]
Perfect :)

File: Exercises/variables-declaration/variables-declaration.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+alert(window.globalV);
+
+// 3. Declare two variables which will be same-names but contain different values.
+const car = 'Volvo';
+console.log(car);
+
+{
+    const car = 'BMW'
+    console.log(car);
+}
```
Comment (Mike-Savin) [inline Exercises/variables-declaration/variables-declaration.js:20]: Have you skipped hoisting task?

Comment (egar347) [reply Exercises/variables-declaration/variables-declaration.js:20]: Yes, as it was mentioned, this topic will be discussed later and I can return to this task again.

**Pull request merged**